seed/static/seed/partials/organization_settings.html

```diff
@@ -20,201 +20,220 @@ <h2><i class="fa fa-cogs"></i> {$:: 'Settings' | translate $}</h2>
             </div>
```

Comment:
Looks like there's a lot of diffs from elements being reorganized. Is it easy to pick out which diffs are "new" and worth a more thorough review?

Reply:
Yeah, the diff is actually super small, I only added a little bit of text and a button:
https://github.com/SEED-platform/seed/pull/2620/files#diff-d72e48d3c27f01f02b8a4f80a211aed92bf92ae06272064861d331995bfa4918R224-R232

Any other changes were primarily adding missing translations